src/test/java/ttt/PromptSpy.java

```diff
@@ -2,6 +2,8 @@
 
 import com.sun.jdi.connect.Connector;
 
+import java.util.List;
+
 import static ttt.Board.*;
```

Comment:
`import com.sun.jdi.connect.Connector`, `import java.util.List` and `import static ttt.Board.*` are not being used.